server/task/task-memory/src/test/java/org/apache/james/task/eventsourcing/TaskAggregateTest.java

```diff
@@ -86,26 +86,28 @@ void givenInProgressTaskEmitEventWhenUpdateAdditionalInformationCommand() {
     }
 
     @Test
-    void givenInProgressTaskWithOneOlderUpdateEventEmitEventWhenUpdateAdditionalInformationCommand() {
+    void givenInProgressTaskWithOneNewerUpdateEventEmitEventWhenUpdateAdditionalInformationCommand() {
         History history = buildHistory(
             eventId -> Created.apply(ID, eventId, new MemoryReferenceWithCounterTask((counter) -> Task.Result.COMPLETED), HOSTNAME),
             eventId -> Started.apply(ID, eventId, HOSTNAME),
             eventId -> AdditionalInformationUpdated.apply(ID, eventId, new MemoryReferenceWithCounterTask.AdditionalInformation(1, timestamp))
         );
         TaskAggregate aggregate = TaskAggregate.fromHistory(ID, history);
-        assertThat(aggregate.update(new MemoryReferenceWithCounterTask.AdditionalInformation(3, timestamp.plusSeconds(3))))
-            .containsExactly(AdditionalInformationUpdated.apply(ID, history.getNextEventId(), new MemoryReferenceWithCounterTask.AdditionalInformation(3, timestamp.plusSeconds(3))));
+        MemoryReferenceWithCounterTask.AdditionalInformation youngerAdditionalInformation = new MemoryReferenceWithCounterTask.AdditionalInformation(3, timestamp.plusSeconds(3));
+        assertThat(aggregate.update(youngerAdditionalInformation))
+            .containsExactly(AdditionalInformationUpdated.apply(ID, history.getNextEventId(), youngerAdditionalInformation));
```

Comment:
we don't really care about the exact content of the returned event.
we could probably simplify the assertion by comparing only the eventId for example